flow/model/record_items.go

```diff
@@ -25,12 +25,14 @@ func ItemsToJSON(items Items) (string, error) {
 
 // encoding/gob cannot encode unexported fields
 type RecordItems struct {
-	ColToVal map[string]types.QValue
+	ColToVal   map[string]types.QValue
+	NoTruncate bool
```

Comment:
This could be size limit instead of bool. Let's go larger than 15MB for non SF connectors

Reply:
In this PR, i only set the callsite in to true, everythign else is false, to maintains backwards compatibility.  if we go larger than 15MB for non SF connectors, we'll break back compatibility, is this okay?